app/api/db/migrations/versions/2022_09_22_11_13_20_964e99759832_create_user_and_role_tables.py

```diff
@@ -1,16 +1,16 @@
-"""first migration
+"""Create user and role tables first migration
```

Comment:
For curiosity, what would go in `migrations/` and not in `migrations/versions`, should we flatten the folder structure to just `migrations`?

Reply:
The migrations folder has alembic configuration and scripts. This is just following the standard structure as generated by it: https://alembic.sqlalchemy.org/en/latest/tutorial.html